fetch_cluster_logs.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# SUMMARY:
+# Helper script that can fetch all logs for any cluster and outputs them in the current working directory.
+# Fetched logs are comprised of journald logs, mesos sandbox logs and diagnostics bundles for all nodes.
+#
+# WARNING:
+# This script should only be used for testing and debugging purposes. Not for long-running production clusters.
+# The reason for this is it uses a public DCOS_ACS_TOKEN and the --insecure flag of the dcos-cli.
+# This script will also not work if the dcos-cli has previously been set up for the specified cluster.
+#
+# USAGE:
+# ./fetch_cluster_logs <master_public_ip>
+
+set +e
+set -x
+
+LOGIN_TOKEN=eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6Ik9UQkVOakZFTWtWQ09VRTRPRVpGTlRNMFJrWXlRa015Tnprd1JrSkVRemRCTWpBM1FqYzVOZyJ9.eyJlbWFpbCI6ImFsYmVydEBiZWtzdGlsLm5ldCIsImVtYWlsX3ZlcmlmaWVkIjp0cnVlLCJpc3MiOiJodHRwczovL2Rjb3MuYXV0aDAuY29tLyIsInN1YiI6Imdvb2dsZS1vYXV0aDJ8MTA5OTY0NDk5MDExMTA4OTA1MDUwIiwiYXVkIjoiM3lGNVRPU3pkbEk0NVExeHNweHplb0dCZTlmTnhtOW0iLCJleHAiOjIwOTA4ODQ5NzQsImlhdCI6MTQ2MDE2NDk3NH0.OxcoJJp06L1z2_41_p65FriEGkPzwFB_0pA9ULCvwvzJ8pJXw9hLbmsx-23aY2f-ydwJ7LSibL9i5NbQSR2riJWTcW4N7tLLCCMeFXKEK4hErN2hyxz71Fl765EjQSO5KD1A-HsOPr3ZZPoGTBjE0-EFtmXkSlHb1T2zd0Z8T5Z2-q96WkFoT6PiEdbrDA-e47LKtRmqsddnPZnp0xmMQdTr2MjpVgvqG7TlRvxDcYc-62rkwQXDNSWsW61FcKfQ-TRIZSf2GS9F9esDF4b5tRtrXcBNaorYa9ql0XAWH5W_ct4ylRNl3vwkYKWa4cmPvOqT5Wlj9Tf0af4lNO40PQ
```

Comment:
This token is _extremely_ security sensitive. It can be used to log into _any_ (open) DC/OS cluster. Where did you get it from, and have you been made aware of its significance?

Reply:
It is not a common DC/OS authentication token, as @bamarni has also said.

Reply:
@jgehrcke I asked Tarun from sdk team how he set up the CLI for their CI and he pointed me this code that they've been using for a while: https://github.com/mesosphere/dcos-commons/blob/master/tools/dcos_login.py#L12

I think the reasoning was that the token doesn't work unless it's the very first time that "dcos cluster setup", but I guess it's still a security hazard for every cluster that was not set up with the dcos cli.
I will remove it.

Reply:
@takirala please read comment above about the token being security sensitive